src/sass/AppBar.scss

```diff
@@ -1,11 +1,45 @@
-.app-bar-line {
-  position: fixed;
-  z-index: 1100;
-  width: 100%;
-  margin: 64px 0 0 -24px;
-  height: 1px;
-  background: #B5B5B5;
-  transform: translate3d(0,0,0);
-  -moz-transform: translate3d(0,0,0);
-  -webkit-transform: translate3d(0,0,0);
+.app-bar {
```

Comment:
The problem here is that I'm overwriting inline style of `material-ui` component so there are two way: add inline styling in `javascript` or add styles with `!important`